client-v2/src/components/FrontsEdit/Edit.tsx

```diff
@@ -45,9 +45,9 @@ const FrontsEditContainer = styled('div')`
 `;
 
 const SingleFrontContainer = styled('div')`
-  flex: 1 1 auto;
+  flex: 1 1 100%;
```

Comment:
Fronts should skew towards filling their container if they can, subject to the max-width constraint applied below.